tools/vue3-webcomponent-wrapper/README.md

```diff
@@ -0,0 +1,27 @@
+# vue3-webcomponent-wrapper
+
+Vue 3 wrapper to convert a Vue component into Web Component. It supports reactive attributes, events & slots.
+
+This is a port of [@vuejs/vue-web-component-wrapper](https://github.com/vuejs/vue-web-component-wrapper) (Official Vue 2 web component wrapper package) to work with Vue 3. And must be deprecated once the official wrapper package  starts supporting Vue 3. Progress of Vue3 support is tracked in this [issue.](https://github.com/vuejs/vue-web-component-wrapper/issues/93)
+
+One main blocker preventing the official wrapper from upgrading was the lack of shadow-root CSS injection in Vue 3 build tooling. As we could live without a shadow dom in Hue this was not an issue and this port was created.
+
+## Usage
+
+    import { createApp, h } from "vue";
+    import wrapper from "vue3-webcomponent-wrapper";
+    import MyComponent from "./components/MyComponent.vue";
+
+    const webComponent = wrapper(MyComponent, createApp, h);
+    window.customElements.define("my-component", webComponent);
```

Comment:
FYI: if I am someone who would be interested in the plugin, usually also having a section `Before` the plugin, aka with the official wrapper, and `After` it is released would make them trust the plugin even more? (i.e. they can very quickly see what code needs to be changed to use the new plugin as well as how to revert back in the future)

Reply:
As of now it's not clear how the official plugin will be used. Will add a note on the before part, aka how it was used in Vue 2.